examples/FrozenLake_psrl.py

```diff
@@ -0,0 +1,101 @@
+import torch
+import pprint
+import argparse
+import numpy as np
+from torch.utils.tensorboard import SummaryWriter
+
+from tianshou.env import SubprocVectorEnv
+from tianshou.trainer import onpolicy_trainer, offpolicy_trainer
+from tianshou.data import Collector, ReplayBuffer
+
+import gym
+from tianshou.env import VectorEnv
+import sys
+import os
+sys.path.append(os.path.abspath(os.path.join(os.path.split(__file__)[0], os.pardir)) + '/tianshou/policy/psrl')
+from psrl import PSRLPolicy, PSRLModel
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--task', type=str, default='FrozenLake-v0')
+    parser.add_argument('--seed', type=int, default=1626)
+    parser.add_argument('--eps-test', type=float, default=0)
+    parser.add_argument('--eps-train', type=float, default=0.1)
+    parser.add_argument('--buffer-size', type=int, default=1)
+    parser.add_argument('--epoch', type=int, default=100)
+    parser.add_argument('--step-per-epoch', type=int, default=100)
+    parser.add_argument('--collect-per-step', type=int, default=100)
+    parser.add_argument('--repeat-per-collect', type=int, default=1)
+    parser.add_argument('--batch-size', type=int, default=64)
+    parser.add_argument('--training-num', type=int, default=8)
+    parser.add_argument('--test-num', type=int, default=1)
+    parser.add_argument('--logdir', type=str, default='log')
+    parser.add_argument('--render', type=float, default=0.)
+    args = parser.parse_known_args()[0]
+    return args
+
+
+def test_psrl(args=get_args()):
+    env = gym.make(args.task)
+    args.state_shape = env.observation_space.shape or env.observation_space.n
+    args.action_shape = env.env.action_space.shape or env.env.action_space.n
+    # train_envs = gym.make(args.task)
+    # train_envs = gym.make(args.task)
+    train_envs = VectorEnv(
+        [lambda: gym.make(args.task) for _ in range(args.training_num)])
+    # test_envs = gym.make(args.task)
+    test_envs = SubprocVectorEnv(
+        [lambda: gym.make(args.task) for _ in range(args.test_num)])
+    # seed
+    np.random.seed(args.seed)
+    torch.manual_seed(args.seed)
+    train_envs.seed(args.seed)
+    test_envs.seed(args.seed)
+    # model
+    n_action = args.action_shape
+    n_state = args.state_shape
+    p_pri = 1e-3 * np.ones((n_action, n_state, n_state))
+    rew_mean = np.zeros((n_state, n_action))
+    rew_std = np.ones((n_state, n_action))
+    model = PSRLModel(p_pri, rew_mean, rew_std)
+    policy = PSRLPolicy(model)
```

Comment:
I think `PSRLModel` can be an inner class, like
```python
policy = PSRLPolicy(p_pri, rew_mean, rew_std, ...)
```
and in the initialization part of PSRLPolicy, refer that part of my comments.